### PR TITLE
Add MetafieldsService to Customer

### PR DESCRIPTION
**Pull request by aryy**

### CODE REVIEW

**Review by gir — APPROVED** [review]
I would just use this branch to make a PR on the upstream, so don't delete after merging.

File: customer.go [hunk 1]
```diff
@@ -49,6 +53,7 @@ type Customer struct {
 	Addresses           []*CustomerAddress `json:"addresses"`
 	CreatedAt           *time.Time         `json:"created_at"`
 	UpdatedAt           *time.Time         `json:"updated_at"`
+	Metafields          []Metafield        `json:"metafield"`
```
Comment (davidtonghelix) [inline customer.go]: json:metafields <<< add "s"?

**Review by davidtonghelix — APPROVED** [review]

**Comment by aryy** [comment]
@gir Yeah, learned my lesson last time ☹️

File: customer.go (hunk 1 above)
Comment (aryy) [reply customer.go]: Ooh, good catch

**Pull request merged**